dev/common/RUN_INTEGRATION_STAGE.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+export HTTPS_PROXY="http://squid.corp.redhat.com:3128"
+export HUB_UPLOAD_SIGNATURES=true
+export IQE_VAULT_GITHUB_TOKEN=${IQE_VAULT_GITHUB_TOKEN}
+export IQE_VAULT_ROLE_ID=${IQE_VAULT_ROLE_ID}
+export IQE_VAULT_SECRET_ID=${IQE_VAULT_SECRET_ID}
+export TESTS_AGAINST_STAGE=true
+export HUB_USE_MOVE_ENDPOINT=true
+export HUB_API_ROOT="https://console.stage.redhat.com/api/automation-hub/"
+export HUB_AUTH_URL="https://sso.stage.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token/"
+
+mkdir -p "${HOME}/venvs"
+venv_path="${HOME}/venvs/ahub-tests-venv"
+python3 -m venv "${venv_path}"
```

Comment:
Any particular reason this script puts the venvs in ${HOME} while common/RUN_INTEGRATION.sh and standalone/RUN_INTEGRATION.sh use /tmp?

The other RUN_INTEGRATION.sh's use virtualenv vs venv. Either should be fine, just wondering if there was a particular need that requires them to be different?  (and/or, should other RUN_INTEGRATION.sh scripts be updated?)